plenum/common/transactions.py

```diff
@@ -8,6 +8,7 @@ class PlenumTransactions(Enum):
     #  Also the numeric constants CANNOT collide with transactions in dependent components.
     NODE = "0"
     NYM = "1"
+    GET_TXN = "106"
```

Comment:
Why don't use "3" as a value? It's assumed that Sovrin-sepcific txns start from 100, and plenum-specific start from 0